src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -413,7 +420,7 @@ protected virtual void SetWorkspaceOptions(TestWorkspace workspace)
                 await VerifyProviderCommitCheckResultsAsync(document2, position, itemToCommit, expectedCodeAfterCommit, commitChar, textTypedSoFar);
             }
         }
-
+        
         private async Task VerifyProviderCommitCheckResultsAsync(
             Document document, int position, string itemToCommit, string expectedCodeAfterCommit, char? commitCharOpt, string textTypedSoFar)
```

Comment:
this entire test method is effectively bogus.  It's "attempting" to validate that post commit, the file will look like what is expected.  However, it's not going through the actual controller commit codepath.  Instead, it's just simulating what it thinks would happen.  This entire test method should be removed, and all tests that depend on it should become CompletoinCommandHandler tests.